products/sle-micro/main.pm

```diff
@@ -74,15 +56,6 @@ if (is_updates_test_repo && !get_var('MAINT_TEST_REPO')) {
 }
 
 return 1 if load_yaml_schedule;
-
-if (is_container_test) {
-    load_boot_from_disk_tests();
-    load_container_tests();
-}
-
-if (is_kernel_test()) {
-    load_kernel_tests();
-    return 1;
-}
+load_tests;
```

Comment:
```suggestion
load_tests; # see main_transactional.pm
```

Let's be nice to our future selves - I would do the same in the other `.pm` files.

Reply:
I actually tried with `main_transactional::load_tests` but for some reason it didn't work...

Reply:
> I actually tried with `main_transactional::load_tests` but for some reason it didn't work...

I was about to suggest that, meh

Reply:
https://openqa.opensuse.org/tests/2928784
`Undefined subroutine &main_micro_alp::load_tests called at os-autoinst-distri-opensuse/products/alp/main.pm line 20.`

Reply:
Done. The reason was obvious, it's not in the same directory, but if I import it, I can use `main_transactional::load_tests` and no need for comment.